manifests/config.pp

```diff
@@ -39,4 +49,56 @@
     group   => $otelcol::config_file_group,
     mode    => $otelcol::config_file_mode,
   }
+
+  $otelcol::receivers.each|String $rname, Hash $rvalue| {
+    if $rvalue['config'] =~ Hash {
+      ensure_resource('Otelcol::Receiver', $rname, $rvalue)
```

Comment:
I think all cases of `ensure_resource` need to use a lowercase value
```suggestion
      ensure_resource('otelcol::receiver', $rname, $rvalue)
```

Reply:
Fixed that for all occurrences